client/blocks/disconnect-jetpack/index.jsx

```diff
@@ -281,10 +281,10 @@ class DisconnectJetpack extends PureComponent {
 					className="disconnect-jetpack__try-rewind disconnect-jetpack__block"
 				>
 					<p className="disconnect-jetpack__highlight">
-						{ translate( 'Experiencing connection issues? Try to go back and rewind your site.' ) }
+						{ translate( 'Experiencing connection issues? Try to go back and restore your site.' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.